OSpackages.sh

```diff
@@ -375,6 +378,34 @@ function FetchCustomRepos {
 
 }
 
+# Prevent Azure-specific failure
+function AzureYumPluginDirCollision {
+  local AZURE_ASSET_TAG
+  local YUM_PROBLEM_DIR
+
+  # The asset-tag value for dmidecode
+  AZURE_ASSET_TAG="7783-7084-3265-9085-8269-3286-77"
```

Comment:
Do we have any links to Azure docs as a source for this value? Thinking ahead to when we look at this in a couple years and wonder how we got the value.

Reply:
Other than StackExchange? Lemme go look.

Reply:
I'll add this URL:

~~~
https://learn.microsoft.com/en-us/azure/automation/troubleshoot/update-agent-issues-linux#dmidecode-check
~~~

Into the script.